Source/WebCore/rendering/style/TextSizeAdjustment.cpp

```diff
@@ -29,6 +29,7 @@
 #if ENABLE(TEXT_AUTOSIZING)
 
 #include "RenderStyle.h"
+#include "RenderStyleInlines.h"
```

Comment:
Same question here

Reply:
After adding the new file I ended up getting compile errors in these files. Not super familiar with how our unified sources pick which files to use but I'm guessing it has something to do with that? Or if there is another way to solve that issue I could do that

Reply:
It makes sense that the inlines would be needed in these implementation files if we are moving code around. It might just be that the UnifiedBuild files got adjusted (since you added a new file) and it no longer gets the benefit of some include that used to be part of the same compilation unit.